library/src/main/java/com/novoda/downloadmanager/lib/DownloadReceiver.java

```diff
@@ -142,7 +148,7 @@ private void sendNotificationClickedIntent(Context context, long[] ids) {
      * {@link DownloadManager#COLUMN_ID}.
      */
     private void openDownload(Context context, long id) {
-        Intent intent = OpenHelper.buildViewIntent(context, id);
+        Intent intent = new OpenHelper(downloadsUriProvider).buildViewIntent(context, id);
```

Comment:
Yes I had a look where and when this was used, and is only used in this line 168 https://github.com/novoda/download-manager/blob/Add-test-plus-refactoring/library/src/main/java/com/novoda/downloadmanager/lib/DownloadReceiver.java, is package protected and is a relatively small class